packages/core/src/index.ts

```diff
@@ -561,6 +561,109 @@ class Connector implements IConnector {
     }
   }
 
+  /**
+   * IW send transaction
+   * @param platform 
+   * @param params 
+   */
+  public async iwSendTransaction (platform: string ,params: any) {
+    if (!this._connected) {
+      throw new Error(ERROR_SESSION_DISCONNECTED)
+    }
+
+    const request = this._formatRequest({
+      method: 'iw_sendTransaction',
+      params: [{
+        platform,
+        params: params
+      }]
+    })
+
+    try {
+      const result = await this._sendCallRequest(request)
+      return result
+    } catch (error) {
+      throw error
+    }
+  }
+
+  /**
+   * IW get wallet address by platform
+   * @param platform 
+   * @param options 
+   */
+  public async iwGetAccounts (platform: string , options: any) {
```

Comment:
@khanm-ibl 
Did this methods return multi accounts ? Or just single account ? The base document have 2 functions is getAddress and getAddresses, please check more details about that.

Reply:
This function will cover 2 case
- options.currentWalletOnly === true // get current wallet's address only
- options.currentWalletOnly is empty or value is false : return all by default
@taipa-ibl let me know your idea, thank!

Reply:
It's good, but the options should be
`options.allWallet === true` for get all wallet accounts
`options.allWallet === false or empty or undefined` for get current wallet only.